src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int width;
+    private int heigth;
+
+    public Rectangle(String color, int width, int heigth) {
+        super(color);
+        this.width = width;
+        this.heigth = heigth;
+    }
+
+    @Override
+    public double getArea() {
+        return (width * heigth) / 2;
```

Comment:
formula is invalid
![image](https://github.com/mate-academy/jv-oop-advanced/assets/73041091/040a2d2f-8a32-4e4b-8af3-5e4041373d4e)